src/rpc/net.cpp

```diff
@@ -113,6 +132,8 @@ static UniValue getpeerinfo(const JSONRPCRequest& request)
                             {RPCResult::Type::STR, "subver", "The string version"},
                             {RPCResult::Type::BOOL, "inbound", "Inbound (true) or Outbound (false)"},
                             {RPCResult::Type::BOOL, "addnode", "Whether connection was due to addnode/-connect or if it was an automatic/inbound connection"},
+                            {RPCResult::Type::NUM, "conn_type", "Connection type between 0 and 5:\n" + ConnTypeList()},
```

Comment:
Why does this matter?

Reply:
To be sure I'm understanding, the suggestion is to drop "between 0 and 5"?

Reply:
No, I'm asking why is this is needed.

Reply:
It's explained in the PR description.